ipykernel/inprocess/manager.py

```diff
@@ -23,7 +24,8 @@ class InProcessKernelManager(KernelManager):
     kernel = Instance('ipykernel.inprocess.ipkernel.InProcessKernel',
                       allow_none=True)
     # the client class for KM.client() shortcut
-    client_class = DottedObjectName('ipykernel.inprocess.BlockingInProcessKernelClient')
+    name = 'ipykernel.inprocess.BlockingInProcessKernelClient'
```

Comment:
It doesn't seem like this should be a `name` attribute on the kernel manager class.